packages/atlas/src/components/TablePaymentsHistory/TablePaymentsHistory.styles.ts

```diff
@@ -33,3 +35,39 @@ export const StyledJoyTokenIcon = styled(JoyTokenIcon)<{ error: boolean }>`
     fill: ${({ error }) => (error ? cVar('colorTextError') : undefined)};
   }
 `
+
+export const JoystreamSvgWrapper = styled.div`
+  height: 31px;
+  width: 31px;
+  border-radius: 50%;
+  display: grid;
+  place-items: center;
+  overflow: hidden;
+  background-color: ${cVar('colorBackgroundMutedAlpha')};
+
+  svg {
+    height: 20px;
+
+    g {
+      fill: ${cVar('colorTextMuted')};
+    }
+  }
+`
```

Comment:
Colors don't match the design: 
![image](https://user-images.githubusercontent.com/51168865/217813809-3e69c777-ec30-46b6-b865-511403449833.png)
Yours: 
![image](https://user-images.githubusercontent.com/51168865/217814025-eb475723-9049-49e3-80ae-867447e80b16.png)

Reply:
@WRadoslaw I know its tiny and annoying, but please check the font color as well.
<img width="530" alt="image" src="https://user-images.githubusercontent.com/13239251/218826879-0f8c3a7b-df87-48b3-b5d3-4f554e437791.png">

Reply:
@WRadoslaw check this one please sir.